File: from-3201-to-3300/3300-minimum-element-after-replacement-with-digit-sum/solution.go
func minElement(nums []int) int {
    minSum := 1000
    for _, num := range nums {
        sum := 0
        for num > 0 {
            sum += num % 10
            num /= 10
        }
        if sum < minSum {
            minSum = sum
        }
    }
    return minSum
}

File: from-3201-to-3300/3300-minimum-element-after-replacement-with-digit-sum/solution.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        minSum = 1000
        for num in nums:
            sum = 0
            while num > 0:
                sum += num % 10
                num //= 10
            if sum < minSum:
                minSum = sum

        return minSum
        